json-endpoints.php

```diff
@@ -3023,3 +3464,36 @@
 		'headers' => array( 'authorization' => 'Bearer YOUR_API_TOKEN' ),
 	),
 ) );
+
+new WPCOM_JSON_API_Publicize_Endpoint( array(
```

Comment:
It looks like we have stopped syncing the file that this endpoint should have been in a while ago. There is the r130158-wpcom changeset by @dsmart that removes the file from sync. Now it seems like @enej has reused that file for this endpoint, and the code is not there, so it fatals.